src/services/goDaddy.ts

```diff
@@ -0,0 +1,8 @@
+const KEY = process.env.KEY
+const SECRET = process.env.SECRET
+
+export default {
+  service: 'https://api.godaddy.com',
+  getDomains: '/v1/domains?statuses=ACTIVE',
+  options: { headers: { Authorization: `sso-key ${KEY}:${SECRET}` } }
+}
```

Comment:
you should be exporting these functions:

1. `export const getDomains...` -> should return a promise that resolves to all the domains.

All goDaddy related services should go here.